### PR TITLE
DCOS-12454: Add Universe link to service picker via MountService

### PR DESCRIPTION
**Pull request by jfurrow**
This PR uses the `MountService` component to add a link to the Universe page from the service picker.

I'm adding `hold merge` until I get approved copy from the docs team: https://mesosphere.atlassian.net/browse/DCOS-12454.

### CODE REVIEW

**Comment by mlunoe** [comment]
Looks good, works great! Let's get this ⛵️ 'ed as soon as copy is approved

File: src/js/pages/UniversePage.js [hunk 1]
```diff
@@ -1,7 +1,21 @@
+import {MountService} from 'foundation-ui';
 import React from 'react';
 
 import Icon from '../components/Icon';
 
+const IntroductoryText = () => {
```
Comment (orlandohohmeier) [inline src/js/pages/UniversePage.js]: I very much like that you used the mount module as this already prepares the extraction of a universe package, but wonder if we should move this into a separate file.

File: src/js/pages/UniversePage.js [hunk 2]
```diff
@@ -1,7 +1,21 @@
...
+  return (
+    <p className="lead tall">
+      Create a containerized service or run a command in one of two ways: use our form to be guided through the correct configuration, or enter your JSON configuration directly. You may also <a href="#/universe">browse the package repository</a>.
```
Comment (orlandohohmeier) [inline src/js/pages/UniversePage.js]: Let's try to no replace the complete copy and only include the "link", as I could image that we eventually want to add more links.

**Comment by jfurrow** [comment]
@orlandohohmeier @mlunoe Sorry, but the design for this has changed so I'm closing this PR until I finish implementing the new design.

File: src/js/pages/UniversePage.js (hunk 2 above)
Comment (jfurrow) [inline src/js/pages/UniversePage.js]: That does make more sense!

**Comment by jfurrow** [comment]
@orlandohohmeier Thanks for the feedback! I implemented it in the new PR for this task: https://github.com/dcos/dcos-ui/pull/1779

**Pull request closed without merging**